EventMixing/inc/Mu2eProductMixer.hh

```diff
@@ -140,6 +167,16 @@ namespace mu2e {
     std::vector<GenOffset> genOffsets_;
 
     void updateSimParticle(SimParticle& particle, SPOffset offset, art::PtrRemapper const& remap);
+
+    typedef std::map<cet::map_vector_key,PhysicalVolumeInfo> VolumeMap;
```

Comment:
Isn't this map \<PhysicalVolumeInfoSingleStage\> ?  It would be less confusing if all the typedefs were defined in PhysicalVolumeInfo.hh and used consistently.  I suggest to move the content of PhysicalVolumeInfoMultiCollection.hh into PhysicalVolumeInfo.hh as well.

Reply:
No, this is a different type optimized for insertions (std::map).   PhysicalVolumeInfoSingleStage is optimized for lookup and storage.